log/logger_stdout_file.go

```diff
@@ -0,0 +1,50 @@
+package log
+
+import (
+	"github.com/op/go-logging"
+	"os"
+)
+
+// StdoutFileLogger outputs logs both to standard output as well as to a given file.
+type StdoutFileLogger struct {
+	basicLogger
+}
+
+func NewStdoutFileLogger(module, logFilePath, logLevel, formatStdout,
+	formatFile string) (*StdoutFileLogger, error) {
+	basicLogger := newBasicLogger(module)
+
+	levelInt, err := logging.LogLevel(logLevel)
+	if err != nil {
+		return nil, err
+	}
+
+	formattedStdoutBackend, err := basicLogger.setupFormattedBackend(os.Stdout, formatStdout)
+	if err != nil {
+		return nil, err
+	}
+	backends := []logging.Backend{formattedStdoutBackend}
+
+	// a non-empty string was provided as argument, which is assumed to be the absolute path
+	// to the file in which to write logs
```

Comment:
Is this comment still valid?

Reply:
Nope, it was left-over from previous implementation when we were checking whether `logFilePath != ""` Thanks.